### PR TITLE
[WIP] Replace deprecated IENetwork with ie.read_network() in Python demos

### PR DESCRIPTION
**Pull request by AnthonyQuantum**

### CODE REVIEW

File: demos/python_demos/face_recognition_demo/face_recognition_demo.py [hunk 1]
```diff
@@ -173,14 +173,13 @@ def __init__(self, args):
...
-        log.info("Loading the model from '%s'" % (model_description_path))
-        assert osp.isfile(model_description_path), \
-            "Model description is not found at '%s'" % (model_description_path)
+        log.info("Loading the model from '%s'" % (model_path))
+        assert osp.isfile(model_path), \
+            "Model description is not found at '%s'" % (model_path)
         assert osp.isfile(model_weights_path), \
             "Model weights are not found at '%s'" % (model_weights_path)
-        model = IENetwork(model_description_path, model_weights_path)
+        model = IECore().read_network(model_path, model_weights_path)
```
Comment (IRDonch) [inline demos/python_demos/face_recognition_demo/face_recognition_demo.py:182]: This should use the core from `self.context`.

File: demos/python_demos/action_recognition/action_recognition_demo/models.py [hunk 2]
```diff
@@ -87,7 +87,7 @@ def __init__(self, model_xml, model_bin, ie_core, target_device, num_requests, b
 
         # Read IR
         print("Reading IR...")
-        self.net = IENetwork(model_xml, model_bin)
+        self.net = IECore().read_network(model_xml, model_bin)
```
Comment (Wovchena) [inline demos/python_demos/action_recognition/action_recognition_demo/models.py:90]: Please, reuse `ie_core`.
And you won't need `from openvino.inference_engine import IECore`.

**Comment by alalek** [comment]
> Merge pull request #

We definitely should not see that in Pull Requests.
Wrong merge direction? Or wrong target branch?

**Comment by AnthonyQuantum** [comment]
> > Merge pull request #
> 
> We definitely should not see that in Pull Requests.
> Wrong merge direction? Or wrong target branch?

Sorry, I've tried to to change the base branch from develop to release, but probably used the wrong command. Will fix soon.

**Comment by alalek** [comment]
BTW, Re-open PR first before force pushing commits to avoid PR locking by GitHub (with all pending reviews)

**Pull request closed without merging**